deployments/sepolia_addresses.json

```diff
@@ -19,6 +19,7 @@
     "MockWBTC": "0x92A2928f5634BEa89A195e7BeCF0f0FEEDAB885b",
     "MockWETH": "0x700868CAbb60e90d77B6588ce072d9859ec8E281",
     "MockcrvUSD": "0x36421d873abCa3E2bE6BB3c819C0CF26374F63b6",
+    "NativeTokenGateway": "0x13B3f65C0e2C64528F678B3C78ccac7341a2A66C",
```

Comment:
I miss the deployment script for this new contract

Reply:
Done https://github.com/VenusProtocol/isolated-pools/pull/361/commits/70f74e2731f005b5fd984ebd53644cd6673fa82f